galleries/examples/lines_bars_and_markers/stackplot_demo.py

```diff
@@ -20,11 +20,11 @@
 # https://population.un.org/wpp/, license: CC BY 3.0 IGO
 year = [1950, 1960, 1970, 1980, 1990, 2000, 2010, 2018]
 population_by_continent = {
-    'africa': [228, 284, 365, 477, 631, 814, 1044, 1275],
-    'americas': [340, 425, 519, 619, 727, 840, 943, 1006],
-    'asia': [1394, 1686, 2120, 2625, 3202, 3714, 4169, 4560],
-    'europe': [220, 253, 276, 295, 310, 303, 294, 293],
-    'oceania': [12, 15, 19, 22, 26, 31, 36, 39],
+    'africa': np.array([.228, .284, .365, .477, .631, .814, 1.044, 1.275]),
+    'americas': np.array([.340, .425, .519, .619, .727, .840, .943, 1.006]),
+    'asia': np.array([1.394, 1.686, 2.120, 2.625, 3.202, 3.714, 4.169, 4.560]),
+    'europe': np.array([.220, .253, .276, .295, .310, .303, .294, .293]),
+    'oceania': np.array([.012, .015, .019, .022, .026, .031, .036, .039]),
```

Comment:
Since you are no longer multiplying, these could be lists rather than arrays, which I think would look a bit cleaner.